src/app/app.component.ts

```diff
@@ -1,10 +1,20 @@
+import { GlobalsService } from './services/globals.service';
 import { Component } from '@angular/core';
+import { Router } from '@angular/router';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'app';
-}
+  title: string
+  redirectPage: string[]
+
+  constructor(public router: Router, gs: GlobalsService) {
+    this.title = gs.APPNAME
+    this.redirectPage = gs.LANDINGPAGE
+
+    router.navigate(this.redirectPage)
```

Comment:
redirectPage is an array.
router.navigate() takes a string, not an array.
right?

Reply:
router.navigate can take a string or an array. You can use an array in case the string is made up of substrings e.g. followers/Zaheer/friends = ['followers', 'Zaheer', 'friends']